applications/nrf5340_audio/src/modules/led.c

```diff
@@ -55,11 +58,13 @@ struct led_unit_cfg {
 static uint8_t leds_num;
 static bool initialized;
 static struct led_unit_cfg led_units[LED_UNIT_MAX];
+static struct led_unit_cfg *led_units_map[LED_UNIT_MAX] = {NULL};
```

Comment:
I may get this later, but here I don't see why we ned the array of pointers.

Reply:
Well we can't use the index for the node and I couldn't see how to pass in the constant. Therefore I map the unit found to the pointer table which then matches the defines for the LED.